components/PageNav.tsx

```diff
@@ -0,0 +1,57 @@
+import Icon from "components/Icon";
+
+interface PageNavProps {
+  currentPage: number;
+  numPages: number;
+  prevPage: () => void;
+  nextPage: () => void;
+  prevDisabled: boolean;
+  nextDisabled: boolean;
+}
+
+const PageNav: React.FunctionComponent<PageNavProps> = ({
+  currentPage,
+  numPages,
+  prevPage,
+  nextPage,
+  prevDisabled,
+  nextDisabled,
+}: PageNavProps) => {
+  return (
+    <div className="mt-16 mb-20 mr-20">
+      <div className="flex flex-row justify-end text-sm">
+        <button
+          type="button"
+          className={
+            prevDisabled
+              ? "flex flex-row items-center opacity-50"
```

Comment:
Instead of applying `opacity-50` here, consider moving the className to Icon instead, and switching the color of the icon depending on the state of `prevDisabled` / `nextDisabled`. 

You can use `className="fill-current stroke-current text-[one of our colors, i.e. unselected or dark]"`

Reply:
I added in those properties that you suggested, except I kept the switching within the button where it is currently because the text also needs to change color